gap/Utils.gi

```diff
@@ -9,6 +13,12 @@ function(field, nrRows, nrCols, entries)
     return ImmutableMatrix(field, m);
 end);
 
+# Return an antidiagonal matrix M with entries as specified by the argument
+# <entries> in the following way:
+#     * If <entries> is a list, the entries of M are, from top right to
+#       bottom left, the entries of that list.
+#     * If <entries> is an integer, the entries of M are all ones and the
+#       number of them is <entries>.
```

Comment:
Could also turn this into an AutoDoc comment attached to `DeclareGlobalFunction("AntidiagonalMat");`